docs/core_docs/scripts/notebook_init_utils.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable */
```

Comment:
Yeah let's cut these if we can't find an automatic way to import them. If we really need something Node specific we can just use `.mdx`

I can also reach out to the Deno team to see if they have suggestions, we have a channel with them.